e2e/src/screens/Android/components/AndroidOpenWithComponent.ts

```diff
@@ -0,0 +1,39 @@
+import { ChainablePromiseElement } from 'webdriverio';
+
+import { AndroidSelectorStrategies } from '../../../Strategies';
+import Utils from '../../../Utils';
+
+class AndroidOpenWithComponent {
+  get openWithMetaMaskQA(): ChainablePromiseElement<WebdriverIO.Element> {
+    return $(
+      Utils.getLocatorPerPlatformAndStrategy({
+        androidLocator: {
+          locator: 'new UiSelector().text("MetaMask-QA")',
```

Comment:
I'd create a constant object with all those strings that are used by selectors etc. So that if they change, it should be easy to change in one place